project/AppDependencies.scala

```diff
@@ -20,39 +20,39 @@ import sbt._
     "uk.gov.hmrc"             %%   "tax-year"                         % taxYearVersion,
-    "uk.gov.hmrc"             %%   "play-partials"                    % playPartialsVersion,
-    "uk.gov.hmrc"             %%   "http-caching-client"              % httpCachingVersion,
-    "uk.gov.hmrc"             %%   "play-conditional-form-mapping"    % playConditionalMappingVersion,
-    "uk.gov.hmrc.mongo"       %%  s"hmrc-mongo$playSuffix"            % hmrcMongoVersion,
+    "uk.gov.hmrc"             %%   s"play-partials$playSuffix"        % playPartialsVersion,
+    "uk.gov.hmrc"             %%   s"http-caching-client$playSuffix"  % httpCachingVersion,
+    "uk.gov.hmrc"             %%   s"play-conditional-form-mapping$playSuffix"    % playConditionalMappingVersion,
+    "uk.gov.hmrc.mongo"       %%   s"hmrc-mongo$playSuffix"           % hmrcMongoVersion,
     "commons-validator"       %    "commons-validator"                % commonsValidatorVersion,
-    "uk.gov.hmrc"             %%   "play-frontend-hmrc"               % playFrontendHmrc
+    "uk.gov.hmrc"             %%   s"play-frontend-hmrc$playSuffix"   % playFrontendHmrc
   )
 
-  val test = Seq(
-    "uk.gov.hmrc"             %%  s"bootstrap-test$playSuffix"        % bootstrapVersion          % "test, it",
-    "org.jsoup"               %   "jsoup"                             % "1.15.4"                  % "test, it",
-    "org.scalatestplus"       %%  "mockito-4-5"                       % s"$scalaTestVersion.0"    % "test, it",
-    "org.scalatestplus.play"  %%  "scalatestplus-play"                % "5.1.0"                   % "test, it",
-    "com.typesafe.play"       %%  "play-test"                         % PlayVersion.current       % "test, it",
-    "com.vladsch.flexmark"    %   "flexmark-all"                      % "0.64.0"                  % "test, it",
-    "org.scalatestplus"       %%  "scalacheck-1-16"                   % s"$scalaTestVersion.0"    % "test, it",
-    "com.github.tomakehurst"  %   "wiremock-jre8-standalone"          % "2.35.0"                  % "it",
-    "uk.gov.hmrc.mongo"       %% s"hmrc-mongo-test$playSuffix"        % hmrcMongoVersion          % "it"
+  val test: Seq[ModuleID] = Seq(
+    "uk.gov.hmrc"             %%  s"bootstrap-test$playSuffix"        % bootstrapVersion          % Test,
+    "org.jsoup"               %   "jsoup"                             % "1.17.2"                  % Test,
+    "org.scalatestplus"       %%  "mockito-4-5"                       % s"$scalaTestVersion.0"    % Test,
+    "org.scalatestplus.play"  %%  "scalatestplus-play"                % "7.0.1"                   % Test,
+    "com.typesafe.play"       %%  "play-test"                         % "2.9.2"                   % Test,
+    "com.vladsch.flexmark"    %   "flexmark-all"                      % "0.64.8"                  % Test,
+    "org.scalatestplus"       %%  "scalacheck-1-16"                   % s"$scalaTestVersion.0"    % Test,
```

Comment:
scalacheck-1-16 should be replaced by "org.scalatestplus" %% "scalacheck-1-17" % "3.2.18.0" % Test